extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -182,7 +182,7 @@ and the user can select a single date. This is the default selection type.
 </amp-date-picker>
 ```
 
-<!-- TODO(cvializ): add image -->
+<img alt="A static single date picker" src="img/amp-date-picker-single-static.png" width="50%">
```

Comment:
As this doc will be imported into ampproject.org, it needs to be valid AMP, otherwise the images won't render.  Can you change the markup like this ([amp-story component does this](https://raw.githubusercontent.com/ampproject/amphtml/master/extensions/amp-story/amp-story.md) and other docs):
```html
<amp-img alt="static single date picker" layout="responsive" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-date-picker/img/amp-date-picker-single-static.png" width="656" height="728">
  <noscript>
    <img alt="static single date picker" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-date-picker/img/amp-date-picker-single-static.png" />
  </noscript>
</amp-img>
```

Otherwise, I'll fix it after the images are added ...

Reply:
oh TIL, thanks!